newIDE/app/src/Profile/EditProfileDialog.js

```diff
@@ -80,7 +133,19 @@ const EditProfileDialog = ({
 
   const donateLinkFormattingError =
     !!donateLink && !simpleUrlRegex.test(donateLink)
-      ? donateLinkFormattingErrorMessage
+      ? profileLinkFormattingErrorMessage
+      : undefined;
+  const personalWebsiteLinkFormattingError =
+    !!personalWebsiteLink && !simpleUrlRegex.test(personalWebsiteLink)
+      ? profileLinkFormattingErrorMessage
+      : undefined;
+  const personalWebsite2LinkFormattingError =
+    !!personalWebsite2Link && !simpleUrlRegex.test(personalWebsite2Link)
+      ? profileLinkFormattingErrorMessage
+      : undefined;
+  const discordServerLinkFormattingError =
+    !!discordServerLink && !simpleDiscordUrlRegex.test(discordServerLink)
+      ? discordServerLinkFormattingErrorMessage
```

Comment:
Should you also test the length of the fields to match the ones in the backend so that the user knows more early that it cannot be too long?
Moreover, I feel like all this validating logic should be in a separate file

Reply:
the textfield has a maxLength defined, so that should be enough!
I'll move them to a separate file for clarity